lib/elements/message.js

```diff
@@ -27,6 +27,9 @@ export default class Message extends React.Component {
     linkContent += ` in ${atom.project.relativizePath(message.filePath)[1]}`
     return <a href="#" className="linter-message-link" onClick={() => visitMessage(message) }>{ linkContent }</a>
   }
+  static getCopyLink(message: Linter$Message) {
+    return <a href="#" className="linter-message-copy-link" onClick={() => atom.clipboard.write(message.text) } />
```

Comment:
Good point. Iirc it's illegal to have both, so `message.text || message.html` should be ok?